calico/openstack/mech_calico.py

```diff
@@ -150,10 +154,16 @@ def __init__(self):
         # Initialize fields for the database object and transport.  We will
         # initialize these properly when we first need them.
         self.db = None
+        self._db_context = None
         self.transport = None
         self._my_pid = None
         self._periodic_resync_greenlet = None
         self._epoch = 0
+        self.in_resync = False
+
+        # Prepare client for accessing status updates in etcd.
+        self._client = None
```

Comment:
I think the split between the mech_calico and transport modules is supposed to be that transport owns the etcd client(s).  why have we created another one in mech_calico?